recipes/mysql-connector-cpp/all/conandata.yml

```diff
@@ -0,0 +1,21 @@
+sources:
+  "9.0.0":
+    url: "https://github.com/mysql/mysql-connector-cpp/archive/refs/tags/9.0.0.zip"
+    sha256: "9d4b90a4efe8861e821136fb3024074d14875749d8b69d1821361b2f2b8bfe55"
+patches:
+  "9.0.0":
+    - patch_file: "patches/9.0.0/0001-override-cmake-policy-version.patch"
+      patch_description: "Set CMake policy version to 3.15, move project() statement"
+      patch_type: "conan"
```

Comment:
```suggestion
```

I really don't see an error when not using this patch. I would avoid if possible.

Reply:
Yeah, looks like the issues were caused by `bootstrap()` rather than the policy version or `project()` call location.